C++/Navio/Common/gpio.cpp

```diff
@@ -137,37 +135,42 @@ void Pin::toggle()
     write(!read());
 }
 
+uint32_t Pin::getPeriBase() const
+{
+    int rpi_version = getRaspberryPiVersion();
+
+    switch (rpi_version) {
+      case 1:
+        return BCM2708_PERI_BASE;
+      case 2:
+        return BCM2709_PERI_BASE;
+      case 3:
+        return BCM2835_PERI_BASE;
+      default:
+        std::cerr << "Unexpected version. Setting up configurations as on RPI2/3" << std::endl;
+        return BCM2709_PERI_BASE;
+    }
+}
```

Comment:
You add this function at this commit and change your current changes in the next one. Won't it be better to swap commits? Fix addresses before this?